.github/ISSUE_TEMPLATE/ui_bug_report_template.yaml

```diff
@@ -21,6 +21,13 @@ body:
     validations:
       required: true
 
+  - type: input
+    validations:
+      required: true
+    attributes:
+      label: MLflow version
+      description: MLflow version (run `mlflow --version`) or commit SHA if you have MLflow installed from source (run `pip freeze | grep mlflow`).
```

Comment:
Created a separate section for mlflow version to prevent users from forgetting to write it + better discoverability.

Reply:
<img width="525" alt="image" src="https://user-images.githubusercontent.com/17039389/169426536-7d60a2e5-6f0a-447d-ab18-d0428a7d7480.png">